extensions/binary.py

```
@@ -0,0 +1,22 @@
+#
+#   BINARY
+#
+
+# BINARY -> NAMESPACE
+binary = ContextVar("binary")
```

Ruff (0.11.9)

Undefined name `ContextVar`

(F821)